contracts/AbstractConference.sol

```diff
@@ -85,18 +97,42 @@ contract AbstractConference is Conference, GroupAdmin {
     /**
      * @dev Withdraws deposit after the event is over.
      */
-    function withdraw() external onlyEnded {
-        require(payoutAmount > 0, 'payout is 0');
-        Participant storage participant = participants[msg.sender];
-        require(participant.addr == msg.sender, 'forbidden access');
-        require(cancelled || isAttended(msg.sender), 'event still active or you did not attend');
-        require(participant.paid == false, 'already withdrawn');
-
-        participant.paid = true;
+    function withdraw() external onlyEnded canWithdraw {
+        participants[msg.sender].paid = true;
         doWithdraw(msg.sender, payoutAmount);
         emit WithdrawEvent(msg.sender, payoutAmount);
     }
 
+    /**
+    * @dev sendAndWithDraw function allows to split _payoutAmount_ among addresses in _addresses_.
+    * 
+    * _addresses_ contains ethereum addresses
+    * _values_ contains the value of eth/dai to give to addresses
+    * 
+    * addresses[i] will receive values[i]
+    * The function emits the event SendAndWithdraw with the following informations:
+    * (addresses, values, participant address, payoutAmount - sum(values), payoutAmount)
+    */
+    function sendAndWithdraw(address payable[] calldata addresses, uint256[] calldata values) external canWithdraw onlyEnded {
+        uint256 sumOfValues = 0;
+        for(uint i = 0; i < values.length; i++) {
```

Comment:
This loop is redundant. Can do this check in the loop below and revert afterwards. Everything will still work as expected without any funds being withdrawn.

Reply:
Done in the last commit.